src/admin-views/editor/panel/ticket.php

```diff
@@ -152,7 +179,135 @@ class="tribe_soft_note ticket_form_right"
 						); ?>
 					</span>
 				</div>
-				<fieldset id="tribe_ticket_provider_wrapper" class="input_block" aria-hidden="true" >
+				<div class="input_block">
+					<label class="ticket_form_label ticket_form_left" for="ticket_start_date"><?php esc_html_e( 'Start sale:', 'event-tickets' ); ?></label>
+					<div class="ticket_form_right">
+						<input
+							autocomplete="off"
+							type="text"
+							class="tribe-datepicker tribe-field-start_date ticket_field"
+							name="ticket_start_date"
+							id="ticket_start_date"
+							value="<?php echo esc_attr( $ticket ? $start_date : null ); ?>"
+							data-validation-type="datepicker"
+							data-validation-is-less-or-equal-to="#ticket_end_date"
+							data-validation-error="<?php echo esc_attr( json_encode( $start_date_errors ) ) ?>"
+						/>
+						<span class="helper-text hide-if-js"><?php esc_html_e( 'YYYY-MM-DD', 'event-tickets' ) ?></span>
+						<span class="datetime_seperator"> <?php esc_html_e( 'at', 'event-tickets' ); ?> </span>
+						<input
+							autocomplete="off"
+							type="text"
+							class="tribe-timepicker tribe-field-start_time ticket_field"
+							name="ticket_start_time"
+							id="ticket_start_time"
+							<?php echo Tribe__View_Helpers::is_24hr_format() ? 'data-format="H:i"' : '' ?>
+							data-step="<?php echo esc_attr( $timepicker_step ); ?>"
+							data-round="<?php echo esc_attr( $timepicker_round ); ?>"
+							value="<?php echo esc_attr( $ticket ? $ticket->start_time : null ); ?>"
+							aria-label="<?php
+							echo esc_html(
+								sprintf(
+									__( '%s start date', 'event-tickets' ),
+									tribe_get_ticket_label_singular( 'input_start_time_aria_label' )
+								)
+							); ?>"
+						/>
+						<span class="helper-text hide-if-js"><?php esc_html_e( 'HH:MM', 'event-tickets' ) ?></span>
+						<span class="dashicons dashicons-editor-help" title="<?php
+						echo esc_html(
+							sprintf(
+								__( 'If you do not set a start sale date, %s will be available immediately.', 'event-tickets' ),
+								tribe_get_ticket_label_plural_lowercase( 'input_start_time_help_text_title' )
+							)
+						); ?>">
```

Comment:
:no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: f485350f4f4248f3a73bc8fd9dcc9cfe</sub>